exercises/practice/triangle/triangle_tests.plt

```diff
@@ -52,13 +62,19 @@ pending :-
     test(sides_may_be_floats, condition(pending)) :-
         triangle((0.5), (0.4), (0.5), "isosceles").
 
+    test(sides_are_a_mix, condition(bonus)) :-
+        triangle(5, 4, 5.0, "isosceles").
```

Comment:
I was thinking of having the bonus tests be in a separate section, together at the end of the file. That way the student can slowly work through the required tests without getting involved in the bonus tests.

Reply:
Good afternoon(GMT),
Yesterday in the evening I was checking the docs better, and as such we can improve this kind of approach with [begin_tests/2](https://www.swi-prolog.org/pldoc/doc_for?object=begin_tests/2).
I've done as you requested.
I did change the indentation, as the previous was getting on my nerves, I can revert that later.

I was just considering if we should just have pending condition, as I'm doing in bonus, for suites, instead of individual tests.